posts/zzz_DO_NOT_EDIT_hello_admiralmetabolic/hello_admiralmetabolic!.qmd

```diff
@@ -0,0 +1,145 @@
+---
+title: "Hello admiralmetabolic!"
+author:
+  - name: Anders Askeland
+  - name: Edoardo Mancini
+description: "We are happy to announce the release of the newest member of the {admiral} family - {admiralmetabolic}!"
+# Note that the date below will be auto-updated when the post is merged.
+date: "2024-12-01"
+# Please do not use any non-default categories.
+# You can find the default categories in the repository README.md
+categories: [ADaM, Community, Technical]
+# Feel free to change the image
+image: "admiralmetabolic.png"
+
+---
+
+<!--------------- typical setup ----------------->
+
+```{r setup, include=FALSE}
+long_slug <- "zzz_DO_NOT_EDIT_hello_admiralmetabolic"
+library(link)
+link::auto()
+```
+
+<!--------------- post begins here ----------------->
+
+[![](admiralmetabolic.png){fig-align="center" width="220"}](https://pharmaverse.github.io/admiralmetabolic/index.html)
+
+# Introduction
+
+The {admiral} team is excited to celebrate the newest addition to the our extension package family with the release {admiralmetabolic} 0.1! This new package focuses on the therapeutic areas of obesity and cardiovascular metabolism, and was borne out of a collaboration between Novo Nordisk and Roche, as well as a number of independent contributors. {admiralmetabolic} addresses a key unmet need in this area given the ever-increasing interest in obesity and cardiovascular metabolism across the industry.
+
+# Package Contents
+
+For this first 0.1 release, {admiralmetabolic} focuses primarily on the obesity indication. The package offers:
+
+* SDTM obesity-specific test data (to be moved to {pharmaversesdtm} at the next release of the package);
+* Two brand-new functions: admiralmetabolic::derive_param_waisthgt() and admiralmetabolic::derive_param_waisthip();
+* Two vignettes for an [obesity vital signs ADaM ADVS](https://pharmaverse.github.io/admiralmetabolic/articles/advs.html) and a [Control of Eating Questionnaire ADaM ADCOEQ](https://pharmaverse.github.io/admiralmetabolic/articles/adcoeq.html);
+* Two template programs for [ADVS](https://github.com/pharmaverse/admiralmetabolic/blob/main/inst/templates/ad_advs.R) and [ADCOEQ](https://github.com/pharmaverse/admiralmetabolic/blob/main/inst/templates/ad_adcoeq.R) matching the corresponding vignettes.
+
+## New Functions
+
+The first function, admiralmetabolic::derive_param_waisthgt(), is a wrapper to admiral::derive_param_computed() and enables users to add derived waist to height ratio parameters to a vital signs ADVS ADaM using waist circumference and height records. Here is an example of the function in action:
```

Comment:
```suggestion
The first function, `admiralmetabolic::derive_param_waisthgt()`, is a wrapper to `admiral::derive_param_computed()` and enables users to add derived waist to height ratio parameters to a vital signs ADVS ADaM using waist circumference and height records. Here is an example of the function in action:
```

Don't know if we back-tick this?

Reply:
Not on the blog because we use auto-linking to link the documentation pages of functions instead.